src/models/models.jl

```diff
@@ -24,7 +24,7 @@ end
 function InfiniteITensorSum(model::Model, s::CelledVector; kwargs...)
   N = length(s)
   H = InfiniteITensorSum(N)
-  tensors = [ITensor(model, s[n], s[n + 1]; kwargs...) for n in 1:N]
+  tensors = [ITensor(model, s[n], s[n + 1]; n = n, kwargs...) for n in 1:N] #support for staggered potentials and dimerized hoppings.
```

Comment:
Please revert this to the original code, since that functionality will be covered by #31.